apps/vercel/frontend/src/locations/ConfigScreen.tsx

```diff
@@ -47,9 +48,13 @@ const ConfigScreen = () => {
       parameters,
       targetState: currentState,
     };
-  }, [parameters, sdk]);
+  }, [parameters, sdk, isTokenValid]);
 
-  const vercelClient = new VercelClient(parameters.vercelAccessToken);
+  useEffect(() => {
+    if (parameters.vercelAccessToken) {
+      setVercelClient(new VercelClient(parameters.vercelAccessToken));
+    }
+  }, [parameters.vercelAccessToken]);
```

Comment:
this was a great suggestion from @rusticpenguin in a pervious PR, to ensure we only instantiate this Client once. This also ensures that we only instantiate when an access token exists, to avoid the error that occurs when communicating with the Vercel API without a token of any sort.